src/client/app/components/admin/CreateUserComponent.tsx

```diff
@@ -37,7 +38,7 @@ export default function CreateUserFormComponent(props: CreateUserFormProps) {
 
 	return (
 		<div className='container-fluid'>
-			<h1 style={titleStyle}> Create User </h1>
+			<h1 style={titleStyle}> <FormattedMessage id='create.user'/> </h1>
```

Comment:
A few lines down has "Email". Could it be translated with "e-mail" as long as doing this file?

Reply:
As mentioned above, fixed the issue regarding email.